File: programmers/Hash/1.js
function solution(participant, completion) {
    var answer = '';
    const table={};
    participant.forEach((el)=>{
        if (table[el]===undefined){
            table[el]=1;
        }
        else{
            table[el]+=1;
        }
    });
    completion.forEach((el)=>{
       if(table[el]===undefined){
           answer=el;
           return
       } 
        table[el]-=1;
        if(table[el]===0){
            delete table[el];
        }
    });
    answer=Object.keys(table)[0];
    return answer;
}

//https://programmers.co.kr/learn/courses/30/lessons/42576

File: programmers/Sort/1.py
#https://programmers.co.kr/learn/courses/30/lessons/42748
def solution(array, commands):
    answer = []
    
    for command in commands:
        s_array=array[command[0]-1:command[1]]
        s_array.sort()
        answer.append(s_array[command[2]-1])
    
    return answer
